### ISSUE TITLE
Hiding status bar

### ISSUE THREAD

**Issue by dfsq** [codesandbox/codesandbox-client#2150]
# 🌈 Feature

Would be nice to be able to hide status bar.

Looks like toggling it in "View › Appearance › Hide/Show Status Bar" as well as changing VSCode setting "Workbench › Status Bar: Visible" both do nothing to the status bar visibility. 

So I wonder if it's implemented at the moment.

Being able to hide it would free some additional space of the screen:

<img width="238" alt="Screen Shot 2019-07-09 at 10 05 09" src="https://user-images.githubusercontent.com/768023/60870402-128f7700-a231-11e9-98b9-a784d68f8e18.png">

**Comment by rubenmoya** [comment]
Hi! I'd love to give this one a try, I guess adding a conditional [here](https://github.com/codesandbox/codesandbox-client/blob/master/packages/app/src/app/pages/Sandbox/Editor/index.js#L134) based on the settings should do it right?

**Comment by rubenmoya** [comment]
I've been able to show/hide it based on the zenMode, but I don't know where to find the `statusBar` visible/hidden setting

**Comment by shiftyp** [comment]
tldr; I wasn't able to find it either, but here's where I looked to save you some time:

1. I looked at the [settings store](https://github.com/codesandbox/codesandbox-client/blob/master/packages/app/src/app/store/providers/SettingsStore.js) to see if there was a key for vscode settings or the status bar.
2. I think the state in that store might come ultimately from [overmind](https://github.com/codesandbox/codesandbox-client/blob/master/packages/app/src/app/overmind/namespaces/preferences/state.ts), and there's no setting there.
3. I looked at the [`VSCodeManager` class](https://github.com/codesandbox/codesandbox-client/blob/master/packages/app/src/app/vscode/index.ts#L33) to see if there was a way to access the vscode settings through it.

Any ideas @SaraVieira or @Saeris?
